pkg/monitoring/rules/alerts/vms.go

```diff
@@ -69,7 +69,7 @@ var (
 		},
 		{
 			Alert: "KubeVirtVMIExcessiveMigrations",
-			Expr:  intstr.FromString("sum by (vmi, namespace) (max_over_time(kubevirt_vmi_migration_succeeded[1d])) >= 12"),
+			Expr:  intstr.FromString("sum by (vmi, namespace) (sum by (vmi, namespace, vmim) (max_over_time(kubevirt_vmi_migration_succeeded[1d]))*0+1) >= 12"),
```

Comment:
nit: use `topk by` instead of `*0+1`:
```go
sum by (vmi, namespace) (topk by (vmi, namespace, vmim) 
(1, max_over_time(kubevirt_vmi_migration_succeeded[1d]))) >= 12
```

Reply:
thanks, the expression has been updated to use `topk`.